### PR TITLE
Add cesium and tangram demo

### PR DESCRIPTION
**Pull request by oterral**
Try to open: https://mf-geoadmin3.int.bgdi.ch/vib/teo_cesiumdemo/templates/tgmap.html
Try to open: https://mf-geoadmin3.int.bgdi.ch/vib/teo_cesiumdemo/templates/glmap.html
Try to open: https://mf-geoadmin3.int.bgdi.ch/vib/teo_cesiumdemo/templates/csmap.html
Try to open: https://mf-geoadmin3.int.bgdi.ch/vib/teo_cesiumdemo/templates/olmap.html

### CODE REVIEW

**Comment by loicgasser** [comment]
coool, what about an index file?

**Comment by loicgasser** [comment]
Good point the common js

File: Makefile [hunk 1]
```diff
@@ -41,14 +41,18 @@ ${INSTALL_DIRECTORY}/devlibs: requirements.txt
...
 
-vib/templates/olmap.mako.html:
 vib/templates/olmap.html: vib/templates/olmap.mako.html
 	${MAKO_CMD} --var "branch_name=${BRANCH_NAME}" $< > $@
 
+vib/templates/csmap.html: vib/templates/csmap.mako.html
+	${MAKO_CMD} --var "branch_name=${BRANCH_NAME}" $< > $@
+
+vib/templates/tgmap.html: vib/templates/tgmap.mako.html
+	${MAKO_CMD} --var "branch_name=${BRANCH_NAME}" $< > $@
```
Comment (loicgasser) [inline Makefile]: yes, I think we could create a function for that in the future.

Comment (oterral) [inline Makefile]: Yes i've thought the same

**Pull request merged**